atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraKeyValueService.java

```diff
@@ -2230,19 +2245,21 @@ private void alterGcAndTombstoneInternal(
                                 tableRef.getQualifiedName(),
                                 configManager.getConfig().schemaMutationTimeoutMillis());
                         log.trace("gc_grace_seconds is set to {} for {}.{}",
-                                gcGraceSeconds, keyspace, tableRef);
+                                SafeArg.of("gcGraceSeconds", gcGraceSeconds), UnsafeArg.of("keyspace", keyspace),
+                                UnsafeArg.of("table", tableRef));
                         log.trace("tombstone_threshold_ratio is set to {} for {}.{}",
-                                tombstoneThresholdRatio, keyspace, tableRef);
+                                SafeArg.of("tombstoneThresholdRatio", tombstoneThresholdRatio),
+                                UnsafeArg.of("keyspace", keyspace), SafeArg.of("table", tableRef));
                     }
                 }
                 return null;
             });
         } catch (Exception e) {
             log.error("Exception encountered while setting gc_grace_seconds:{} and tombstone_threshold:{} for {}.{}",
-                    gcGraceSeconds,
-                    tombstoneThresholdRatio,
-                    keyspace,
-                    tableRef,
+                    SafeArg.of("gcGraceSeconds", gcGraceSeconds),
+                    SafeArg.of("tombstoneThresholdRatio", tombstoneThresholdRatio),
+                    UnsafeArg.of("keyspace", keyspace),
+                    UnsafeArg.of("table", tableRef),
                     e);
```

Comment:
Not marking is Unsafe, which is correct (exception messages are unsafe according to the spec).